EIPS/eip-1418.md

```diff
@@ -9,6 +9,14 @@ category: Core
 created: 2018-09-16
 ---
 
+# Simple Summary
+
+At each block, deduct an amount of value from every account based on the quantity of storage used by that account.
+
+# Abstract
+
+The most naive implementation would be to simply loop through every account on each block and deduct a certain fee. We show that a better implementation could achieve reasonable performance. Also we review practical considerations of switching to a fee-based rent system.
```

Comment:
This would require an immense amount of data reads and updates if you'd want also update the state trie. As you state in your abstract, there might be a way to do this faster, but if the state trie isn't updated at every block, you lose the ability to prove balances on an account.

Reply:
You didn't read the whole paper! The answers are below.

Reply:
Here's an extra commit, a97c77d4ff827a9d3b3abc0672a3077f677192fa

Reply:
Oh you're right. The location where I entered that comment was in GitHub Files Changes tab and I didn't see the rest of the document. Mistook it for another EIP that had less information. I suppose the extra commit wasn't really necessary (it might actually complicate things) - I should have just continued reading the rest before commenting.

Reply:
:-p I think we're okay to merge now and have many more changes coming.